components/board/numato/board.go

```diff
@@ -365,9 +367,6 @@ func (ar *analogReader) Close(ctx context.Context) error {
 
 func connect(ctx context.Context, name resource.Name, conf *Config, logger logging.Logger) (board.Board, error) {
 	pins := conf.Pins
-	if pins <= 0 {
-		return nil, errors.New("numato board needs pins set in attributes")
```

Comment:
A small bugfix I snuck in here: I moved this from `connect` to `Config.Validate`. There are literally no configs in the database that don't specify the pin count, and I wouldn't expect that moving it from the constructor to the config validation should change anything. but validation is the right place to have this.